packages/components/src/utils/math.js

```diff
@@ -6,7 +6,7 @@ import { clamp } from 'lodash';
 /**
  * Parses and retrieves a number value.
  *
- * @param {any} value The incoming value.
+ * @param {unknown} value The incoming value.
```

Comment:
Out of curiosity, why this change? I'm not fully up on when to use `unknown`. I guess it makes semantic sense here but what's the exact reason?

Reply:
Glad you asked 🙂 

`any` and `unknown` are both boxes that we can put anything into. The difference is that `any` satisfies every other type, whereas `unknown` is the opposite. It satisfies no other type and requires us to be careful to use it. In code that's like:

```ts
// Anything _satisfies_ both types `any` and `unknown`
var __a: any = null
var __a: any = undefined
var __a: any = ['foo']
var __a: any = 0
const a: any = new Set(['apples', 'bananas'])

var __u: unknown = null
var __u: unknown = undefined
var __u: unknown = ['foo']
var __u: unknown = 0
const u: unknown = new Set(['apples', 'bananas'])


// Type system catches this. `unknown` doesn't satisfy anything! Phew…
u.toLocaleLowerCase(); // TYPE ERROR!
// We can't call set methods either, we said we didn't know anything >:/
u.has('foo'); // TYPE ERROR!
// Typically we'll use type guards to be safe with our `unknown` types :D
u instanceof Set && u.has('apples'); // No type error 👍

// `any` type satisfies everything. It's dangerous, as if we turn off the type system completely!!
a.toLocaleLowerCase() // Oops… Set doesn't implement that! We hit s runtime error "TypeError: u.toLocaleLowerCase is not a function" 💥
```